index.js

```diff
@@ -127,25 +145,20 @@ const drawCommand = (message) => {
   });
 };
 
-/*
-const inventoryCommand = () => {
-
-if user draws a card
-store name of card to firebase under the key cardName
-when the same user draws a second card, store the name of the second card under the previous draw within cardName
-continue to do this to all additional cards
-
-user{
-  userID{
-    lastDrawTime: xxxx
-    username: ""
-    cardName: "Blue Eyes White Dragon", "Red Eyes Black Dragon"
-  }
-}
+const inventoryCommand = (message) => {
```

Comment:
By the way don't make this change, but just for reference, this is what the minimal version of this function might look like, balancing readability with concision. There is a big gray area in how much to value concision over clarity, and I think most people would settle on something more similar to what you have. But within that gray area, this is one alternative that is more concise but less clear:

```js
const inventoryCommand = (message) => {
  db.ref("users").once("value", (snapshot) => {
    const userData = snapshot.val();
    const inventoryList = Object.values(userData[message.author.id]).map(
      (item) => "•  " + item.name
    );
    message.reply("your inventory:\n" + inventory.join("\n"));
  });
}
```